tests/sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/geckoview_version_v1/test_aggregation/org_mozilla_fennec_aurora.metrics.yaml

```diff
@@ -0,0 +1,6 @@
+- client_info:
```

Comment:
Can you add a test case for when there are more than 5 pings each for geckoversion negative, between 0 and 100, and >100 at the same timestamp?  My concern is that geckoview version is a string so doing MAX will put 99.0.0 > 100.0.0

Reply:
Yep, I'm working through fixing the tests since they broken in that last change I made.